src/community/design-system-day/session-information.md

```diff
@@ -1,268 +1,7 @@
-
-### Workshop: Using task lists in your services
-
-Frankie Roberto
-3pm to 4:30pm
-
-This session takes a detailed look at the new task list component, and will show you how to implement the wider task list pattern within your service through a live demo.
-
-This will include how to:
-
-- use the new task list component
-- split up a complex application into smaller tasks
-- give the user control over when tasks are marked as completed
-- allow users to review all their tasks before submitting
-
-You'll also have the opportunity to share screenshots from your service, and get feedback from others on whether and how a task list pattern might work for it.
-
-### The iPhone state – simplicity through complexity
-
-Gordon Guthrie
-3pm to 4:30pm
-
-iPhones work but no one individual knows how. They are technically, legally and linguistically sophisticated but just work globally. The modern digital state must embrace simplicity through management of complexity - taking systems view of state capability from manifesto and think tank, through parliamentary oversight to design, delivery and in-service.
-
-### Workshop: How participatory research can help us co-design more ethically
-
-Jane Martin and Betty Mwema
-3pm to 4:30pm
-
-{{ govukWarningText({
-  text: "This session has been cancelled."
-}) }}
-
-In this session, we will be looking at how the field of participation has lived a long time and how it has much to offer to co-design, ethics, equity and justice.
-
-We'll look at how we can develop co-design practice with an ethics-based approach that's safe and informed.
-
-### Designing for underserved audiences
-
-Valentine Makhouleen
-3pm to 4:30pm
-
-What do gender-affirming care providers, childhood mental health experts, reindeer herders and life science researchers have in common? Aside from being underserved, they are leading the charge in changing the world. Their budgets are small, requirements large and needs endless. How do we design meaningful tools for those changing the world?
-
-### From chaos to clarity – unlocking the potential of design systems
-
-Hassan Nawaz
-3pm to 4:30pm
-
-Experience the evolution from chaos to clarity within design systems. See their potential unlocked, creating a united future for experiences that yield impact.
-
-As we embark on this immersive session, prepare to gain actionable insights that empower you to harness the untapped potential of design systems. Together, let's unlock a united future where the synergy of creativity and clarity gives rise to impactful experiences that resonate and endure.
-
-### Fostering Collaboration in design systems
-
-James Carleton
-3pm to 4:30pm
-
-Design systems work best when they are collaborative and flexible instead of a strict set of fixed rules. I would like to share my experience in building a community that encourages collaboration and contribution to our DS.
+To find out about Design System Day, see the [Design System Day](/community/design-system-day/) page.
```

Comment:
Maybe only say "Design System Day" once?

Just need a new line after it too for our EditorConfig settings (there's a Prettier warning)

```suggestion
See the [Design System Day](/community/design-system-day/) page to find out more.

```

Reply:
Ah, you star!